config/webpack.config.js

```diff
@@ -14,12 +16,15 @@ var production =
   process.env.RAILS_ENV === 'production' ||
   process.env.NODE_ENV === 'production';
 
-var config = {
-  entry: {
-    // Sources are expected to live in $app_root/webpack
-    bundle: './webpack/assets/javascripts/bundle.js'
-  },
+var plugins = JSON.parse(execSync('./script/plugin_webpack_directories.rb'));
 
+var config = {
+  entry: Object.assign(
+    {
+      bundle: './webpack/assets/javascripts/bundle.js'
+    },
+    plugins.entries
```

Comment:
Out of curiosity, where is the part that adds '-core.js' to the name of the config entries? I see `ansiblecore.js` here for example, https://gist.github.com/7b39b650c625aba5d7f4216fa7a0c61d (on foreman_ansible I just have `webpack/index.js`)